javascript-loops/main.js

```diff
@@ -0,0 +1,82 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+ function getNumbersToTen(){
+   var numbers = [];
+   var currentNumber = 1;
+
+   var i = 0
+   while(i < 10){
+     numbers[i] = currentNumber ;
+    currentNumber = currentNumber + 1;
+    i++;
+   }
+   return numbers;
+ }
+
+ function getEvenNumbersToTwenty(){
+   var evenNumbers = [];
+   var currentNumber = 2;
+
+   var i = 0;
+   while(i < 10){
+    evenNumbers[i] = currentNumber;
+    currentNumber += 2;
+    i++;
+   }
+   return evenNumbers;
+ }
+
+ function repeatWord(word, times){
+   var count = 1;
+   var repeated = " ";
+
+   while(count <= times){
+    repeated = repeated + " " + word;
+    count++;
+   }
+   return repeated;
+ }
+
+function logEachCharacter(string){
+  for(var i = 0; i < string.length; i++){
+    console.log(string[i]);
+  }
+}
+
+function doubleAll(numbers){
+  var double = [];
+  for(var i = 0; i < numbers.length; i++){
+    double[i] = numbers[i] * 2;
+  }
+  return double;
+}
+
+function getKeys(object){
+  var keys = [];
+
+  var i = 0;
+  for(var prop in object){
+
+    keys[i] = prop;
```

Comment:
Instead of adding values to an array by setting indexes directly, it would be preferable to use the `push` method 👍